index.html

```diff
@@ -30,17 +30,20 @@ <h1 class="head-name">Szabi Sreiber</h1>
     </header>
 
     <main class="main">
-      <article id="profile">
-        <h2>Profile</h2>
-        <p>
-          Hi everybody, I am Szabi Student of Coding Bootcamp Europe. I am
-          intresting long time for <strong>WEB-Development</strong>, and just
-          because I am here in sorty time learn much as possible
-        </p>
-      </article>
-
-      <article id="experience">
+      <div class="oben">
```

Comment:
Szabi, warum div?

Reply:
just because :) but in this place it was necessary because of Grid